Pythonwin/pywin/mfc/dialog.py

```diff
@@ -90,6 +90,7 @@ def items(self):
     def values(self):
         return list(self.data.values())
 
+    # TODO
     # XXX - needs py3k work!
     def has_key(self, key):
         return key in self.data
```

Comment:
How should this be updated to remove old python support code?

Reply:
I really can't recall